catalog/neon-prod-earthengine/projects_neon-prod-earthengine_assets_HSI_REFL_001.jsonnet

```diff
@@ -0,0 +1,3426 @@
+          },
+          {
+            value: 1,
+            color: '474747',
+            description: 'shadow',
+          },
+          {
+            value: 2,
+            color: '9db8d1',
+            description: 'thin cirrus (water)',
+          },
+          {
+            value: 3,
+            color: '5a6978',
+            description: 'medium cirrus (water)',
+          },
+          {
+            value: 4,
+            color: '3d4a57',
+            description: 'thick cirrus (water)',
+          },
+          {
+            value: 5,
+            color: '5d7554',
+            description: 'land (clear)',
+          },
+          {
+            value: 6,
+            color: '454d42',
+            description: 'saturated',
+          },
+          {
+            value: 7,
+            color: 'f7f7f7',
+            description: 'snow/ice',
+          },
+          {
+            value: 8,
+            color: 'd5e3d3',
+            description: 'thin cirrus (land)',
+          },
+          {
+            value: 9,
+            color: '9da89b',
+            description: 'medium cirrus (land)',
+          },
+          {
+            value: 10,
+            color: '5a6359',
+            description: 'thick cirrus (land)',
+          },
+          {
+            value: 11,
+            color: 'ced4cd',
+            description: 'thin haze (land)',
+          },
+          {
+            value: 12,
+            color: '8c918c',
+            description: 'medium haze (land)',
+          },
+          {
+            value: 13,
+            color: 'ceddde',
+            description: 'thin haze/glint (water)',
+          },
+          {
+            value: 14,
+            color: '849596',
+            description: 'med. haze/glint (water)',
+          },
+          {
+            value: 15,
+            color: '98a39b',
+            description: 'cloud (land)',
+          },
+          {
+            value: 16,
+            color: '9899a3',
+            description: 'cloud (water)',
+          },
+          {
+            value: 17,
+            color: '135e9c',
+            description: 'water',
+          },          
+          {
+            value: 18,
+            color: 'e1e8ed',
+            description: 'cirrus cloud',
+          },
+          {
+            value: 19,
+            color: '9ea7ad',
+            description: 'cirrus cloud (thick)',
+          },
+          {
+            value: 20,
+            color: 'dffa11',
+            description: 'bright',
+          },
+          {
+            value: 21,
+            color: '383734',
+            description: 'topographic shadow',
+          },
+          {
+            value: 22,
+            color: '4f4f4e',
+            description: 'cloud (building) shadow',
+          },
+      ]
+     },
+     {
+      name: 'B432',
+      'gee:units': 'deg',
+      description: 'Illumination Factor used as input to ATCOR'
+     },
+     {
+      name: 'B433',
+      'gee:units': 'm',
```

Comment:
Should we use units.meter, like https://github.com/NEONScience/earthengine-catalog/blob/dd5e2037ef00cfe7ef93ff86cf638ee41bacaac7/catalog/USGS/USGS_NED.jsonnet#L67 ?